controllers/release/release_adapter.go

```diff
@@ -161,8 +161,11 @@ func (a *Adapter) EnsureReleasePipelineStatusIsTracked() (results.OperationResul
 
 // createReleasePipelineRun creates and returns a new release PipelineRun. The new PipelineRun will include owner
 // annotations, so it triggers Release reconciles whenever it changes.
```

Comment:
I'd like a quick couple words about release strategy in the comment too

Reply:
Will change it.